front-end/src/index.css

```diff
@@ -0,0 +1,2 @@
+
+ 
```

Comment:
Below you have 500+ files because you have bootstrap installed as a dependency for your root directory - delete the node modules form your root folder to save on this